### PR TITLE
Neweva

### PR DESCRIPTION
**Pull request by tekilaz1**

### CODE REVIEW

File: calculator.html [hunk 1]
```diff
@@ -11,15 +11,35 @@
...
      </h1>
+
+
+        
        
         <div class="wrap">
-            <form name="cal">
-            <input type="text" name="display">
+        <form name="cal">
+            <input type="text" name="display" readonly>
```
Comment (dorimedini) [inline calculator.html]: In the next commit you delete the `readonly` attribute; no need to add it here. Please remove it

**Review by AwetKebedom — COMMENTED** [review]
great tekila you can merge it now!!!!!

File: calculator.html [hunk 2]
```diff
@@ -70,7 +70,7 @@ <h1>
 
             <input type="button" value="0" onclick="cal.display.value+='0'">
             <input type="button" value="." onclick="cal.display.value+='.'">
-            <input type="button" value="=" onclick="cal.display.value=eval(cal.display.value)">
+            <input type="button" value="=" onclick="cal.display.value=math.eval(cal.display.value)">
```
Comment (dorimedini) [inline calculator.html:73]: Where is the `<script>` tag that imports the `math.js` library?

**Review by dorimedini — CHANGES_REQUESTED** [review]
Nice! Still need some changes though, see my comments

**Review by Tesfit12 — COMMENTED** [review]
This is very such an amazing project, i have no comments on it.  <<<<<=========

**Pull request merged**